components/input/private/inner-input.jsx

```diff
@@ -297,8 +297,10 @@ const InnerInput = (props) => {
 					style={props.style}
 					tabIndex={props.tabIndex}
 					type={props.type}
-					value={props.value}
-					defaultValue={props.defaultValue}
+					/* A form element should not have both value and defaultValue props. */
+					{...(props.defaultValue
```

Comment:
should this check for `props.defaultValue !== undefined` to ensure falsey (but intended) values are passed along?

Reply:
also should this prioritize `(props.value)` over `(props.defaultValue)`?